docs/web-servers/nginx/how-to-install-nginx-ubuntu-18-04/index.md

```diff
@@ -21,44 +23,71 @@ external_resources:
 
 NGINX is an open source web server with powerful load balancing, reverse proxy, and caching features. It was [initially designed](https://www.nginx.com/resources/glossary/nginx/) to solve scaling and concurrency problems with existing web servers. Its event-based, asynchronous architecture has made it one of the most popular and best-performing web servers available.
 
+## Before You Begin
+
+1.  Set up your Linode in the [Getting Started](/docs/getting-started/) and [Securing your Server](/docs/security/securing-your-server/) guides.
+
+1.  If you want a custom domain name for your site, you can set this up using our [DNS Manager](/docs/platform/manager/dns-manager/) guide.
+
+    - Don't forget to update your `/etc/hosts` file with the public IP and your site's fully qualified domain name as explained in the [Update Your System's hosts File](/docs/getting-started/#update-your-system-s-hosts-file) section of the [Getting Started](/docs/getting-started/) guide.
+
+    {{< content "limited-user-note-shortguide" >}}
+
 ## Install NGINX
 
-Currently, the best way to install NGINX on Ubuntu 18.04 is to use the version included in Ubuntu's repositories:
+Currently, the best way to install NGINX on Ubuntu 18.04 LTS is to use the version included in Ubuntu's repositories:
 
     sudo apt update
     sudo apt install nginx
 
-## Configure NGINX
+## Add a Basic Site
 
-### Add Basic Site
+1.  Create a new directory for your site.
 
-NGINX site-specific configuration files are kept in `/etc/nginx/sites-available` and symlinked into  `/etc/nginx/sites-enabled/`. Generally you will want to create a separate original file in the `sites-available` directory for each domain or subdomain you will be hosting, and then set up a symlink in the `sites-enabled` directory.
+        sudo mkdir /var/www/example.com
+
+1.  NGINX site-specific configuration files are kept in `/etc/nginx/sites-available` and symlinked to  `/etc/nginx/sites-enabled/`. Generally, you will create a new file containing a [*server block*](https://www.nginx.com/resources/wiki/start/topics/examples/server_blocks/) in the `sites-available` directory for each domain or subdomain you will be hosting. Then, you will set up a symlink to your files in the `sites-enabled` directory.
```

Comment:
I think this paragraph needs to be about the location of site files, as this paragraph is (rightly) included in the section below it.

Reply:
Fixed